presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -145,7 +147,7 @@
 public class PlanPrinter
 {
     private final PlanRepresentation representation;
-    private final FunctionAndTypeManager functionAndTypeManager;
+    private FunctionAndTypeManager functionAndTypeManager;
```

Comment:
Why removing the final keyword? It won't be reassigned anywhere else. 
Also, if the access modifiers are changed in any case, make sure they are grouped properly.

Reply:
no this variable should be marked as `final` and shouldn't be removed, fixed.